dpc-go/dpc-api/pkg/api/v2/organization.go

```diff
@@ -0,0 +1,44 @@
+package v2
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"go.uber.org/zap"
+
+	"github.com/CMSgov/dpc/pkg/api/client"
+)
+
+func OrganizationCtx(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		organizationID := chi.URLParam(r, "organizationID")
+		ctx := context.WithValue(r.Context(), "organization", organizationID)
+		next.ServeHTTP(w, r.WithContext(ctx))
+	})
+}
+
+type OrganizationController struct {
+	attributionClient *client.AttributionClient
+}
+
+func NewOrganizationController(config *client.AttributionConfig) *OrganizationController {
+	attributionClient := client.NewAttributionClient(config)
```

Comment:
The Go community likes short variable names when the scope is small like this
```suggestion
	c := client.NewAttributionClient(config)
```